setup.py

```diff
@@ -325,6 +325,7 @@ def make_release_tree(self, base_dir, files):
     ],
     install_requires=[
         "cycler>=0.10",
+        "fonttools>=4.13.0,<5.0",
```

Comment:
This minimum version needs to go in `requirements/testing/minver.txt` so it's actually tested.

Reply:
hmm, the tests still fail with `NoModuleFound` error even after adding it to minver.txt
Any idea?